Many files #include generated UI-files with the path where they are
generated by CMake 3.8; this breaks autouic in CMake 3.9, which looks
at the actual source path and generates the files in a different
location. All upstreamed in different forms for the forthcoming 0.9.4.

--- src/GUI/Library/ImportFolderDialog/GUI_ImportFolder.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Library/ImportFolderDialog/GUI_ImportFolder.h
@@ -21,7 +21,7 @@
 #ifndef GUIIMPORTFOLDER_H_
 #define GUIIMPORTFOLDER_H_
 
-#include "GUI/Library/ui_GUI_ImportFolder.h"
+#include "ui_GUI_ImportFolder.h"
 #include "GUI/Helper/SayonaraWidget/SayonaraDialog.h"
 #include "Components/Library/Importer/LibraryImporter.h"
 
--- src/GUI/Library/InfoBox/GUI_LibraryInfoBox.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Library/InfoBox/GUI_LibraryInfoBox.h
@@ -25,7 +25,7 @@
 #ifndef GUILIBRARYINFOBOX_H_
 #define GUILIBRARYINFOBOX_H_
 
-#include "GUI/Library/ui_GUI_LibraryInfoBox.h"
+#include "ui_GUI_LibraryInfoBox.h"
 #include "GUI/Helper/SayonaraWidget/SayonaraDialog.h"
 
 
--- src/GUI/Playlist/BottomBar/GUI_PlaylistBottomBar.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Playlist/BottomBar/GUI_PlaylistBottomBar.h
@@ -24,7 +24,7 @@
 #define PLAYLISTBOTTOMBAR_H
 
 #include "GUI/Helper/SayonaraWidget/SayonaraWidget.h"
-#include "GUI/Playlist/ui_GUI_PlaylistBottomBar.h"
+#include "ui_GUI_PlaylistBottomBar.h"
 
 #include "Helper/Shutdown/Shutdown.h"
 #include "Helper/Playlist/PlaylistMode.h"
--- src/GUI/Preferences/Broadcast/GUI_BroadcastSetup.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/Broadcast/GUI_BroadcastSetup.h
@@ -23,7 +23,7 @@
 #ifndef GUI_BROADCASTSETUP_H
 #define GUI_BROADCASTSETUP_H
 
-#include "GUI/Preferences/ui_GUI_BroadcastSetup.h"
+#include "ui_GUI_BroadcastSetup.h"
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
 
 class GUI_BroadcastSetup :
--- src/GUI/Preferences/Fonts/GUI_FontConfig.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/Fonts/GUI_FontConfig.h
@@ -24,7 +24,7 @@
 #define GUI_FONTCONFIG_H
 
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
-#include "GUI/Preferences/ui_GUI_FontConfig.h"
+#include "ui_GUI_FontConfig.h"
 
 #include <QFontDatabase>
 #include <QFont>
--- src/GUI/Preferences/Language/GUI_LanguageChooser.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/Language/GUI_LanguageChooser.h
@@ -26,7 +26,7 @@
 #include <QMap>
 #include <QShowEvent>
 
-#include "GUI/Preferences/ui_GUI_LanguageChooser.h"
+#include "ui_GUI_LanguageChooser.h"
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
 
 
--- src/GUI/Preferences/LastFM/GUI_LastFM.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/LastFM/GUI_LastFM.h
@@ -29,7 +29,7 @@
 #ifndef GUI_LASTFM_H_
 #define GUI_LASTFM_H_
 
-#include "GUI/Preferences/ui_GUI_LastFM.h"
+#include "ui_GUI_LastFM.h"
 
 #include "Components/StreamPlugins/LastFM/LastFM.h"
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
--- src/GUI/Preferences/LibraryPreferences/GUI_LibraryPreferences.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/LibraryPreferences/GUI_LibraryPreferences.h
@@ -22,7 +22,7 @@
 #define GUI_LIBRARYPREFERENCES_H
 
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
-#include "GUI/Preferences/ui_GUI_LibraryPreferences.h"
+#include "ui_GUI_LibraryPreferences.h"
 
 
 class GUI_LibraryPreferences :
--- src/GUI/Preferences/Notifications/GUI_Notifications.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/Notifications/GUI_Notifications.h
@@ -25,7 +25,7 @@
 
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
 #include "Interfaces/Notification/NotificationHandler.h"
-#include "GUI/Preferences/ui_GUI_Notifications.h"
+#include "ui_GUI_Notifications.h"
 
 
 class GUI_Notifications :
--- src/GUI/Preferences/PlayerPreferences/GUI_PlayerPreferences.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/PlayerPreferences/GUI_PlayerPreferences.h
@@ -24,7 +24,7 @@
 #define GUI_PLAYERPREFERENCES_H
 
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
-#include "GUI/Preferences/ui_GUI_PlayerPreferences.h"
+#include "ui_GUI_PlayerPreferences.h"
 
 
 class GUI_PlayerPreferences :
--- src/GUI/Preferences/PlaylistPreferences/GUI_PlaylistPreferences.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/PlaylistPreferences/GUI_PlaylistPreferences.h
@@ -28,7 +28,7 @@
 
 
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
-#include "GUI/Preferences/ui_GUI_PlaylistPreferences.h"
+#include "ui_GUI_PlaylistPreferences.h"
 
 
 class GUI_PlaylistPreferences :
--- src/GUI/Preferences/PreferenceDialog/GUI_PreferenceDialog.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/PreferenceDialog/GUI_PreferenceDialog.h
@@ -25,7 +25,7 @@
 
 #include <QDialog>
 #include <QList>
-#include "GUI/Preferences/ui_GUI_PreferenceDialog.h"
+#include "ui_GUI_PreferenceDialog.h"
 #include "GUI/Helper/SayonaraWidget/SayonaraWidget.h"
 #include "Interfaces/PreferenceDialog/PreferenceDialogInterface.h"
 
--- src/GUI/Preferences/RemoteControl/GUI_RemoteControl.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/RemoteControl/GUI_RemoteControl.h
@@ -25,7 +25,7 @@
 #ifndef GUISOCKETSETUP_H_
 #define GUISOCKETSETUP_H_
 
-#include "GUI/Preferences/ui_GUI_RemoteControl.h"
+#include "ui_GUI_RemoteControl.h"
 
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
 
--- src/GUI/Preferences/Shortcuts/GUI_ShortcutEntry.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/Shortcuts/GUI_ShortcutEntry.h
@@ -24,7 +24,7 @@
 
 #include "GUI/Helper/SayonaraWidget/SayonaraWidget.h"
 #include "GUI/Helper/Shortcuts/ShortcutHandler.h"
-#include "GUI/Preferences/ui_GUI_ShortcutEntry.h"
+#include "ui_GUI_ShortcutEntry.h"
 
 
 /**
--- src/GUI/Preferences/Shortcuts/GUI_Shortcuts.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/Shortcuts/GUI_Shortcuts.h
@@ -24,7 +24,7 @@
 #define GUI_SHORTCUTS_H
 
 
-#include "GUI/Preferences/ui_GUI_Shortcuts.h"
+#include "ui_GUI_Shortcuts.h"
 #include "GUI/Helper/Shortcuts/ShortcutHandler.h"
 
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
--- src/GUI/Preferences/StreamRecorder/GUI_StreamRecorder.h.orig	2016-10-30 11:53:25 UTC
+++ src/GUI/Preferences/StreamRecorder/GUI_StreamRecorder.h
@@ -25,7 +25,7 @@
 #ifndef GUI_STREAMRECORDER_H_
 #define GUI_STREAMRECORDER_H_
 
-#include "GUI/Preferences/ui_GUI_StreamRecorder.h"
+#include "ui_GUI_StreamRecorder.h"
 #include "Interfaces/PreferenceDialog/PreferenceWidgetInterface.h"
 
 class GUI_StreamRecorder :
